pdc_client/plugins/component.py

```diff
@@ -70,6 +70,14 @@ def global_component_info(self, args, global_component_id=None):
         global_component_id = global_component_id or args.global_component_id
         global_component = self.client['global-components'][global_component_id]._()
 
+        component_contacts = get_paged(self.client['global-component-contacts']._, component=global_component['name'])
+        contacts = []
+        if component_contacts:
```

Comment:
This code looks like almost same as what you did for `release-component-contacts`. How about abstract it out to a single function avoiding code duplicated?